### PR TITLE
For publication lists, sort by and display publication year.

### LINKED ISSUE #188: Add publication date to the publication title on the researcher detail page (where pubs are linked and listed)

### PR DESCRIPTION
**Pull request by justinlittman**
closes #188

### CODE REVIEW

File: app/models/solr_document.rb [hunk 1]
```diff
@@ -73,7 +73,7 @@ def linked_publications(params)
     response, _deprecated_stuff = search_service.search_results
     docs = response.documents
     tuples = docs.map do |doc|
-      "<li><a href=\"#{search_link(doc.id)}\">#{doc.title}</a></li>"
+      "<li><a href=\"#{search_link(doc.id)}\">#{doc.title}</a> (#{doc['created_year_isim'].first})</li>"
```
Comment (jcoyne) [inline app/models/solr_document.rb]: how about `doc.first('created_year_isim')` instead.  Does that work?

File: spec/search_builders/publication_search_builder_spec.rb [hunk 2]
```diff
@@ -16,6 +16,7 @@
       it 'filters for id and type' do
         expect(solr_params[:fq]).to eq ['authors_ssim:"123abc"', 'type_ssi:Publication']
         expect(solr_params[:rows]).to eq 1000
+        expect(solr_params[:sort]).to eq 'created_year_isim desc'
```
Comment (jcoyne) [inline spec/search_builders/publication_search_builder_spec.rb]: Sorting by a multivalued field works now?  That's been a troublespot in the past.

File: app/models/solr_document.rb (hunk 1 above)
Comment (justinlittman) [reply app/models/solr_document.rb]: Works for me.

File: spec/search_builders/publication_search_builder_spec.rb (hunk 2 above)
Comment (justinlittman) [reply spec/search_builders/publication_search_builder_spec.rb]: It appears to ...
screen shot 2018-11-09 at 1 48 51 pm

**Pull request merged**